src/box/applier.cc

```diff
@@ -1995,29 +1997,30 @@ applier_subscribe(struct applier *applier)
 		 * In case of successful subscribe, the server
 		 * responds with its current vclock.
 		 *
-		 * Tarantool > 2.1.1 also sends its cluster id to
+		 * Tarantool > 2.1.1 also sends its replicaset UUID to
 		 * the replica, and replica has to check whether
-		 * its and master's cluster ids match.
+		 * its and master's replicaset UUIDs match.
 		 */
-		xrow_decode_subscribe_response_xc(&row, &cluster_id,
-					&applier->remote_vclock_at_subscribe);
+		struct subscribe_response rsp;
+		memset(&rsp, 0, sizeof(rsp));
```

Comment:
Let's move the memset to `xrow_decode_subscribe_response`. And do similarly for other `xrow_decode_...` helpers.

It would be nicer to have it in `xrow_decode_replication_request`, but it seems this can't be done now.

Reply:
Thanks, done. In `replication_request` it can be done, but it wouldn't be much better - pass `sizeof` as one of the params to make `memset` inside.